capsules/extra/src/lsm303agr.rs

```diff
@@ -409,11 +409,7 @@ impl<I: i2c::I2CDevice> i2c::I2CClient for Lsm303agrI2C<'_, I> {
     fn command_complete(&self, buffer: &'static mut [u8], status: Result<(), i2c::Error>) {
         match self.state.get() {
             State::IsPresent => {
-                let present = if status == Ok(()) && buffer[0] == 60 {
-                    true
-                } else {
-                    false
-                };
+                let present = status == Ok(()) && buffer[0] == 60;
```

Comment:
I personally prefer `let present = status.is_ok() && buffer[0] == 60` but this is fine too.
This also appears in `capsules/extra/src/lsm303dlhc.rs` and `capsules/extra/src/mlx90614.rs`.